app/views/admin/projects/_project.html.erb

```diff
@@ -3,7 +3,6 @@
   <td><%= link_to project.name, project %></td>
   <td>
     <%= link_to "Edit", edit_project_path(project) %>
-    |
```

Comment:
I have no idea how did I delete this line. And even less how did I miss the change. 
I reverted it.